ui/js/actions/content.js

```diff
@@ -187,55 +188,57 @@ export function doUpdateLoadStatus(uri, outpoint) {
             type: types.DOWNLOADING_PROGRESSED,
             data: {
               uri,
-              outpoint,
               fileInfo,
               progress,
+              sd_hash,
             },
           });
 
           const totalProgress = selectTotalDownloadProgress(getState());
           setProgressBar(totalProgress);
 
           setTimeout(() => {
-            dispatch(doUpdateLoadStatus(uri, outpoint));
+            dispatch(doUpdateLoadStatus(uri, outpoint, sd_hash));
           }, DOWNLOAD_POLL_INTERVAL);
         }
       });
   };
 }
 
-export function doStartDownload(uri, outpoint) {
+export function doStartDownload(uri, outpoint, sd_hash) {
```

Comment:
Can this be re-written to only use `sd_hash` rather than both `outpoint` and `sd_hash`?

Reply:
TL;DR I'm not sure.
It went a bit too deep inside `lbry.js` and it seemed to be intertwined with other stuff(such as pending publishes, dummy claims). Overwhelmed by the complications involved I couldn't muster up enough courage to mess with it.